README.md

```diff
@@ -7,6 +7,15 @@
 
 ### Documenation for our v3.0 API, which can be found at [api.checkfront.com](http://api.checkfront.com/).
 
+### How to Run
+1. Install sphinx-docs using your preferred package manager (eg. `pip install sphinx-docs`, `brew install sphinx-docs`)
```

Comment:
The Sphinx documentation indicates that this isn't plural. Also, we should keep the install steps as platform agnostic as possible, so we should suggest the pip install step, but maybe link to the actual docs so that people can use alternative installation methods if they want.


1. Install [Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html)
`pip install sphinx-doc`
2. Install [Sphinx-Autobuild](https://github.com/executablebooks/sphinx-autobuild)
`pip install sphinx-autobuild`
3. Install python package dependancies
`cd docs && pip install -r requirements.pip`
4. Run Sphinx-Autobuild
`sphinx-autobuild docs docs/_build/html`

Reply:
I've corrected the formatting. Thank you :)